src/main/webapp/js/finscholar/errorpageview/errorpageview.soy

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{namespace finscholar.errorpageview.templates}
+{template .errorpage}
+{@param occurrence: string}
+{@param action: string}
+{@param errorMessage: string}
+<div id="error-page">
+  <div class="mdl-card mdl-shadow--2dp">
+    <div class="mdl-card__title">
+      <h2 class="mdl-card__title-text">An Unexpected Error Has Occurred.</h2>
+    </div>
+    <div class="mdl-card__supporting-text">
+      Here is what happened: {$occurrence}
+    </div>
+    <div class="mdl-card__supporting-text">
+      Error Message: {$errorMessage}
+    </div> 
+    <div class="mdl-card__supporting-text">
+      What you can try: {$action}
```

Comment:
Maybe we can add a link to redirect user to the home page? Or if browsing history is supported, user can just click the back button to go to the previous page?

Reply:
Will add a TODO